tmail-backend/apps/distributed/src/test/resources/blob.properties

```diff
@@ -0,0 +1,89 @@
+# ============================================= BlobStore Implementation ==================================
```

Comment:
This file is likely not needed...

Reply:
No
If we don't have it, `DistributedServerTest/EncryptedDistributedServerTest` will throw exception `org.apache.commons.configuration2.ex.ConfigurationException: blob configuration was not found` 
**Reason**: Old DistributedServerTest uses BlobStoreConfiguration with `cassandra`. Then in the new BlobStoreConfiguration, we removed `implementation` option, keep only ObjectStorage.